src/main/java/com/direwolf20/buildinggadgets/common/utils/helpers/InventoryHelper.java

```diff
@@ -25,13 +25,6 @@
 
 import java.util.*;
 
-/**
- * @MajorTuvok
- * Please check of the changes with the Sets as I've had to to some guess work
- *
- * @MichaelHillcox
```

Comment:
My comment needs to stay as it's still something needed to be done

Reply:
I removed it because it conflicted with JavaDoc and thought it was old... I'll put it into a normal multiline then.